cvs diff: Diffing solenv
cvs diff: Diffing solenv/bin
cvs diff: Diffing solenv/bin/modules
cvs diff: Diffing solenv/bin/modules/installer
cvs diff: Diffing solenv/bin/modules/installer/windows
cvs diff: Diffing solenv/bin/modules/packager
cvs diff: Diffing solenv/bin/modules/par2script
cvs diff: Diffing solenv/bin/modules/pre2par
cvs diff: Diffing solenv/config
cvs diff: Diffing solenv/inc
Index: solenv/inc/unxlngppc4.mk
===================================================================
RCS file: /cvs/tools/solenv/inc/unxlngppc4.mk,v
retrieving revision 1.27
retrieving revision 1.27.56.1
diff -u -u -p -r1.27 -r1.27.56.1
--- solenv/inc/unxlngppc4.mk	18 Jun 2007 16:33:51 -0000	1.27
+++ solenv/inc/unxlngppc4.mk	18 Jul 2007 09:16:38 -0000	1.27.56.1
@@ -82,7 +82,7 @@ CFLAGSEXCEPTIONS=-fexceptions -fno-enfor
 CFLAGS_NO_EXCEPTIONS=-fno-exceptions
 
 CFLAGSCXX= -fsigned-char -pipe -frtti $(ARCH_FLAGS)
-
+PICSWITCH:=-fPIC
 
 # Compiler flags for compiling static object in multi threaded environment with graphical user interface
 CFLAGSOBJGUIMT=$(PICSWITCH)
cvs diff: Diffing solenv/inc/startup
cvs diff: Diffing solenv/inc/startup/AIX
cvs diff: Diffing solenv/inc/startup/FREEBSD
cvs diff: Diffing solenv/inc/startup/HPUX
cvs diff: Diffing solenv/inc/startup/IRIX
cvs diff: Diffing solenv/inc/startup/LINUX
cvs diff: Diffing solenv/inc/startup/MACOSX
cvs diff: Diffing solenv/inc/startup/NETBSD
cvs diff: Diffing solenv/inc/startup/SOLARIS
cvs diff: Diffing solenv/inc/startup/UNIX
cvs diff: Diffing solenv/inc/startup/wnt
cvs diff: Diffing solenv/prj
cvs diff: Diffing solenv/src
cvs diff: Diffing solenv/unxmacxp
cvs diff: Diffing solenv/unxmacxp/inc
